scenarios/kubernetes_verify.py

```diff
@@ -79,9 +79,10 @@ def get_git_cache(k8s):
 
 def branch_to_tag(branch):
     verify_branch = re.match(r'release-(\d+\.\d+)', branch)
-    if not verify_branch:
-        return VERSION_TAG['default']
-    return VERSION_TAG[verify_branch.group(1)]
+    key = 'default'
+    if verify_branch and verify_branch.lastindex == 1:
```

Comment:
When would `verify_branch` be true and not `verify_branch.lastindex == 1`? That is the case that produces a different outcome now, but I don't see how the regexp would match without the capture group matching.

Reply:
er right, overly defensive after the current "it matches but we fail to look up the key". will drop the `and ...`

Reply:
done